dev-libs/libxml++2/libxml++2-2.42.3.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="C++ wrapper for libxml"
+DESCRIPTION="libxml++ (a.k.a. libxmlplusplus) provides a C++ interface to XML files. \
+It uses libxml2 to access the XML files.
+
+libxml++-2.6, libxml++-3.0, libxml++-4.0 and libxml++-5.0 are different \
+parallel-installable ABIs. This package contains libxml++-2.6."
+HOMEPAGE="https://libxmlplusplus.sourceforge.net/"
+COPYRIGHT="2004-2024 LibXML++"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/libxml++/2.42/libxml++-$portVersion.tar.xz"
+CHECKSUM_SHA256="74b95302e24dbebc56e97048e86ad0a4121fc82a43e58d381fbe1d380e8eba04"
+SOURCE_FILENAME="libxml++-$portVersion.tar.bz2"
+SOURCE_DIR="libxml++-$portVersion"
+
+ARCHITECTURES="?all x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	libxml++2$secondaryArchSuffix = $portVersion
+	lib:libxml++_2.6$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libglibmm$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglibmm$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+#	graphviz$secondaryArchSuffix # Optional, for documentation?
+#	libxslt_tools$secondaryArchSuffix # Optional
+#	mm_common$secondaryArchSuffix # Optional
+	"
+BUILD_PREREQUIRES="
+#	cmd:dblatex$secondaryArchSuffix # Optional, for documentation?
+#	cmd:doxygen$secondaryArchSuffix # Optional, for documentation?
```

Comment:
doxygen is also without $secondaryArchSuffix and should be accompanied with cmd:dot
given that, documentation fails to build so dblatex and doxygen can be removed

Reply:
Documentation is actually disabled by default, it can be enabled in the build process. Should I do that?

Reply:
As mentioned above, it fails to build, you can check it, but having them in when not working makes them obsolete.